apps/drupal/templates/field/image.html.twig

```diff
@@ -0,0 +1,25 @@
+{#
+/**
+ * @file
+ * Theme override of an image.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the img tag.
+ * - style_name: (optional) The name of the image style applied.
+ *   We're using as a class below. Set you style in Drupal to a
+ *   Bootstrap style for automatic styling.
+ *   See https://getbootstrap.com/docs/4.0/content/images/
+ *
+ * @see template_preprocess_image()
+ */
+#}
+
+{% set image_classes = [
+  'm-3',
+  'img',
+  'img-responsive',
+  style_name|clean_class,
+  'float-right'
```

Comment:
Same with this. This will affect ALL image fields, not just on this view mode/template. Need to move these positional things to a different hook.